Protocol.md

```diff
@@ -135,17 +123,16 @@ The receiving node MAY reject the contract if:
   - the `contract_info` is missing relevant events.
   - it does not want to use the oracle(s) specified in `oracle_info`.
   - `total_collateral_satoshis` is too small.
-  - `feerate_per_vb` is too small.
-  - `feerate_per_vb` is too large.
+  - the fee rate of the `partial_funding_psbt` is insufficient
 
```

Comment:
There should probably be a requirement that the `PSBT_IN_WITNESS_UTXO ` field is filled for each input

and the `PSBT_IN_REDEEM_SCRIPT ` and `PSBT_IN_WITNESS_SCRIPT ` are filled for inputs that have redeem/witness scripts.

Without these you cannot correctly calculate the fee.

Reply:
https://github.com/discreetlogcontracts/dlcspecs/pull/82/commits/6589f5e2ef4543ef3ff67bf14b8526a67cfb9c23